industrial_robot_client/src/joint_trajectory_streamer.cpp

```diff
@@ -69,19 +69,20 @@ void JointTrajectoryStreamer::jointTrajectoryCB(const trajectory_msgs::JointTraj
 
   // read current state value (should be atomic)
   int state = this->state_;
-
+  
   ROS_DEBUG("Current state is: %d", state);
+
   if (TransferStates::IDLE != state)
   {
-    if (msg->points.empty())
-      ROS_INFO("Empty trajectory received, canceling current trajectory");
-    else
-      ROS_ERROR("Trajectory splicing not yet implemented, stopping current motion.");
+    // if (msg->points.empty())
```

Comment:
Is there a reason that the you commented out these messages?

Reply:
This was probably because the final client considered any red ERROR message they saw on the console as a reportable event.  I didn’t realize this had made it into the top level.

Reply:
I fixed this in the current version of the PR.